app/helpers/dropdown_helper.rb

```diff
@@ -15,6 +15,30 @@ def organisation_options_for_select(selected = nil)
     options_for_select(organisation_options, selected)
   end
 
+  def topic_options_for_select(selected = nil)
+    topic_option_attributes = ->(topic) do
+      [
+        *topic.links.map { |link| link.target.title.squish },
```

Comment:
Do we only ever expect 1 and only 1 link here? If so, I'd prefer something like `first` rather than the splat operator to make that clear.

Reply:
Better yet, could we just ask for `topic.parent` if that's what we want? It's quite hard to understand what's going on in this lambda, especially before reading the scope from below.

Reply:
Addressed in a0665f1
cc/ @andrewgarner